contracts/upgrade/ModuleRegistry.sol

```diff
@@ -46,27 +42,6 @@ contract ModuleRegistry is Owned {
         emit ModuleDeRegistered(_module);
     }
 
-        /**
```

Comment:
I think we should not modify the ModuleRegistry contract at all as the ` UpgraderRegistered` event is used by the client to load the list of possible upgraders from the blockchain. Also, all current modules point to the actual ModuleRegistry so updating it will make the upgrades more complicated.